download-multiple-secure-files-task/task.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Microsoft/azure-pipelines-task-lib/master/tasks.schema.json",
+  "id": "2b1ae9cd-0a4f-4059-b563-61ae159f7496",
+  "name": "DownloadMultipleSecureFiles",
+  "friendlyName": "Download multiple secure files",
+  "description": "Download multiple secure files to the agent machine",
+  "helpMarkDown": "",
+  "category": "Utility",
+  "author": "Adrian Ochmann",
+  "version": {
+    "Major": 1,
+    "Minor": 1,
+    "Patch": 4
+  },
+  "visibility": ["Build"],
+  "runsOn": ["Agent", "DeploymentGroup", "Server", "MachineGroup"],
+  "minimumAgentVersion": "2.116.0",
+  "instanceNameFormat": "Download multiple secure files",
+  "inputs": [
+    {
+      "name": "secureFiles",
+      "type": "secureFile",
+      "label": "Secure Files",
+      "defaultValue": "",
+      "required": true,
+      "helpMarkDown": "The file names or GUID's of the secure files to download into the agent machine. The files will be deleted after the pipeline runs. Each filename needs to be placed as comma separated values."
+    },
+    {
+      "name": "retryCount",
+      "type": "string",
+      "label": "Retry Count",
+      "defaultValue": "5",
+      "required": false,
+      "helpMarkDown": "Optional number of times to retry downloading a secure file if the download fails."
```

Comment:
I'd reword it to:

Number of times to retry downloading a secure files if the download fails (optional, default: 5).